Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -460,7 +464,7 @@ public struct DocumentationNode {
                         range: range,
                         identifier: "org.swift.docc.UnsupportedDocCommentDirective",
                         summary: "Directives are not supported in symbol source documentation",
```

Comment:
It would be good to expand on these diagnostics in the future. But for now, maybe we should just make it a little more specific?

```suggestion
                        summary: "The \(directive.name.singleQuoted) directive is not supported in symbol source documentation",
```

That way the diagnostic isn't incorrectly stating that _all_ directives are unsupported.